src/encoded/static/components/item.js

```diff
@@ -148,7 +146,7 @@ globals.listing_titles.fallback = function () {
     return title;
 };
 
-
+// Removed this functionality for now...
```

Comment:
just a note on style, adding a TODO at the start of comments that imply more action to do in the future is helpful so that the code base can be grepped to determine outstanding technical debt that needs to be taken care of.